installers/installer.sh

```diff
@@ -176,37 +176,32 @@ function __adjustTCPKeepalive ()
 
 function __get_datadisk()
 {
-    local env=$1
-    # AWS is /mnt/datadisk whereas azure and gcp are /datadisk
-    if [[ "$env" == "AWS" ]]; then
-        echo "/mnt/datadisk"
-    else 
-        echo "/datadisk"
-    fi
+    echo "/datadisk"
 }
 
 function __formatDataDisk ()
 {
     local os=$1
     local env=$2
     local sync_gateway=$3
-    if [[ "$env" == "AWS" && "$sync_gateway" -eq "0" ]]; then
-        __log_debug "AWS: Formatting data disk"
-        DEVICE=/dev/sdk
-        MOUNTPOINT=/mnt/datadisk
-        mkfs -t ext4 ${DEVICE}
+    local disk=$4
+    MOUNTPOINT="/datadisk"
+    if [[ "$sync_gateway" -eq "0" ]]; then
```

Comment:
Why is disk option dependent on sync gateway?

Reply:
Sync gateway doesn't store data on disk.